commands/call.js

```diff
@@ -51,7 +57,19 @@ async function scheduleFunctionCall(options) {
 
     const near = await connect(options);
     const account = await near.account(options.accountId);
-    const parsedArgs = options.base64 ? Buffer.from(options.args, 'base64') : JSON.parse(options.args || '{}');
+    let parsedArgs;
+    // load args rare order first
+    if (options.base64file) {
+        try {
+            let fileargs = await readFileSync(options.base64file);
+            parsedArgs = Buffer.from(fileargs, 'base64');
+            console.log(`Loaded base64 args file, size ${parsedArgs.length}`);
+        } catch (e) {
+            console.log('Could not load base64 file!');
```

Comment:
Shouldn't this throw so as exit?  Currently all errors throw and you end up with a large noisy help message, but there are plans to refactor how errors are handled.

Reply:
Good call, changed!